### ISSUE TITLE
Contribute this to Atom?

### ISSUE THREAD

**Issue by daviwil** [harmsk/atomic-management#40]
Hi @harmsk!  Nice work on this package.  We were actually planning to add this functionality as a core Atom feature.  Since you've already done the work here, would you be interested in contributing the functionality to the core Atom codebase?  I'd certainly be happy to help guide you through it if you're interested.

Thanks a lot!

**Comment by harmsk** [comment]
TL;DR: Yes, I would like this in core!

One of my requirements for the project team was that this be integrated into core. However, I later relaxed this requirement after the project team decided that a package had a greater chance of success.

Unfortunately, not being in core has lead to some undesirable side effects. Notably, there is no control over package loading order in Atom. What this means in practice is Atom loads with your global config and then after the Atomic Management package loads, it changes your settings again. This is slow and is jarring to see your project config settings set really late in the window loading process. This is also really undesirable for packages that you want disabled for the open project. Currently, all of your packages are enabled and loaded via the global config only to then have some packages disabled in the project config a moment later. Having this feature in Atom core would make it easier to address these issues.

**Comment by malarinv** [comment]
@harmsk @daviwil is it being added to core? any update? Thanks.

**Comment by harmsk** [comment]
@malarinv @daviwil I would really like this in core. 

@daviwil please help guide me through the process.

**Comment by malarinv** [comment]
@daviwil bump

**Comment by aviatesk** [comment]
@harmsk @daviwil Still not migrated into core packages ??

**Comment by laike9m** [comment]
Any update?

**Comment by daviwil** [comment]
Hi all, my apologies, I'm no longer involved in the Atom project so I don't know whether this is something that would make it into Atom Core at any point.  If folks are still interested in it, I'd recommend petitioning the Atom development team by filing an issue on the [main repository](https://github.com/atom/atom).  Thanks!
